apps/retrospective/services/sprint_task.go

```diff
@@ -53,6 +54,11 @@ func (service SprintTaskService) List(
 		if task.IsTrackerTask {
 			task.URL = connection.GetTaskUrl(task.Key)
 		}
+		var participants_slice = strings.Split(task.TaskParticipants, ", ")
```

Comment:
Please, user lowerCamelCase for variables,
```suggestion
 		var participantsSlice = strings.Split(task.TaskParticipants, ", ")
```

Please add comments here!!

Reply:
Done.